docs/src/components/HeaderNavBar.tsx

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import * as React from 'react';
+import NextLink, { LinkProps } from 'next/link';
+import { styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Popper from '@material-ui/core/Popper';
+import Paper from '@material-ui/core/Paper';
+import Fade from '@material-ui/core/Fade';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import SvgProductCore from 'docs/src/icons/SvgProductCore';
+import SvgProductAdvanced from 'docs/src/icons/SvgProductAdvanced';
+import SvgProductTemplates from 'docs/src/icons/SvgProductTemplates';
+import SvgProductDesign from 'docs/src/icons/SvgProductDesign';
+import SvgMuiX from 'docs/src/icons/SvgMuiX';
+
+const Navigation = styled('nav')(({ theme }) => ({
+  '& ul': {
+    padding: 0,
+    margin: 0,
+    listStyle: 'none',
+    display: 'flex',
+  },
+  '& li': {
+    color: '#5a6978',
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '0.875rem',
+    fontWeight: 'bold',
+    '& > a, & > div': {
+      display: 'inline-block',
+      color: 'inherit',
+      textDecoration: 'none',
+      padding: theme.spacing(1),
+      borderRadius: theme.shape.borderRadius,
+      '&:hover, &:focus': {
+        backgroundColor: '#f3f6f9',
+      },
+    },
+    '& > div': {
+      cursor: 'default',
+    },
+  },
+}));
+
+const PRODUCT_IDS = ['product-core', 'product-advanced', 'product-templates', 'product-design'];
+
+type ProductSubMenuProps = {
+  icon: React.ReactElement;
+  name: React.ReactNode;
+  description: React.ReactNode;
+  href: LinkProps['href'];
+} & Omit<JSX.IntrinsicElements['a'], 'ref'>;
+
+const ProductSubMenu = React.forwardRef<HTMLAnchorElement, ProductSubMenuProps>(
+  function ProductSubMenu({ icon, name, description, href, ...props }, ref) {
+    return (
+      <NextLink href={href} passHref>
+        <Box
+          component="a"
+          ref={ref}
+          display="flex"
+          alignItems="center"
+          py={2}
+          sx={{ '&:hover, &:focus': { backgroundColor: '#f3f6f9', outline: 'none' } }}
+          {...props}
+        >
+          <Box px={2}>{icon}</Box>
+          <Box>
+            <Typography color="#3D4752" fontWeight="bold">
+              {name}
+            </Typography>
+            <Typography color="#5A6978">{description}</Typography>
+            <Box
```

Comment:
I have [just removed](https://rebranding-demo.webflow.io/) the `Learn more` link at the Products popover from the navbar. I think it's not needed here, only on the Product section.